app/models/concerns/track_saved_changes.rb

```diff
@@ -0,0 +1,48 @@
+# https://github.com/ccmcbeck/after-commit
+module TrackSavedChanges
+  extend ActiveSupport::Concern
+
+  included do
+    # expose the details if consumer wants to do more
+    # attr_reader :ts_saved_changes_history, :ts_saved_changes_unfiltered
```

Comment:
Do we still need this line?

Reply:
We do not. Technically it can be removed, but I'd rather leave it in case it becomes useful for a future feature.